<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Hayi Nukman</author>
 <class>DialogSiteGrabber</class>
 <widget class="QDialog" name="DialogSiteGrabber">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>293</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>595</width>
    <height>316</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbBack">
       <property name="text">
        <string>&lt; Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbNext">
       <property name="text">
        <string>Next &gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbRun">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="pageProject">
      <layout class="QGridLayout" name="gridLayout_13">
       <item row="0" column="0" rowspan="3">
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="image.qrc">:/image/icon/big/Internet Options.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;Grabber Project&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Project Name</string>
           </property>
           <property name="buddy">
            <cstring>lePName</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lePName"/>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Project Dir     </string>
           </property>
           <property name="buddy">
            <cstring>lePDir</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lePDir"/>
         </item>
         <item>
          <widget class="QToolButton" name="pbPDir">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_12">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Project Type</string>
           </property>
           <property name="buddy">
            <cstring>cbPType</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbPType">
           <item>
            <property name="text">
             <string>HTTP (Offline Browsing)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FTP</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Custom</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>URL     </string>
           </property>
           <property name="buddy">
            <cstring>lePURL</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lePURL"/>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>37</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageOfflineBrowsing">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Customize for Offline Browsing Project</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QCheckBox" name="cbMirror">
            <property name="toolTip">
             <string>Use recursion rule as Mirroring rule.
- Turn on Recursion
- Turn on Time Stamping
- Sets infinite recursion depth
- Keeps FTP directory listings</string>
            </property>
            <property name="text">
             <string>Use Mirroring Rule</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="label_6">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>101</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>Level depth</string>
             </property>
             <property name="buddy">
              <cstring>sbLevelDepthOBP</cstring>
             </property>
            </widget>
            <widget class="QSpinBox" name="sbLevelDepthOBP">
             <property name="geometry">
              <rect>
               <x>140</x>
               <y>10</y>
               <width>63</width>
               <height>25</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>Set maximum recursion depth. (Use 0 to set Infinite depth)</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="cbConvertLinkOBP">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>40</y>
               <width>181</width>
               <height>25</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>After the download is complete, convert the links in the
document to make them suitable for local viewing.  This
affects not only the visible hyperlinks, but any part of
the document that links to external content, such as
embedded images, links to style sheets, hyperlinks to
non-HTML content, etc.</string>
             </property>
             <property name="whatsThis">
              <string>After the download is complete, convert the links in the document to make them suitable for local viewing.  This affects not only the visible hyperlinks, but any part of the document that links to external content, such as embedded images, links to style sheets, hyperlinks to non-HTML content, etc.</string>
             </property>
             <property name="text">
              <string>Convert Link</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="cbCreateBackupOBP">
             <property name="geometry">
              <rect>
               <x>210</x>
               <y>40</y>
               <width>161</width>
               <height>25</height>
              </rect>
             </property>
             <property name="text">
              <string>Create Backup</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="cbFollowFTPOBP">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>70</y>
               <width>121</width>
               <height>25</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>Follow FTP links from HTML documents.  Without this option, ignore all the FTP links.</string>
             </property>
             <property name="text">
              <string>Follow FTP</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="cbFollowRelativeOBP">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>100</y>
               <width>241</width>
               <height>25</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>Follow relative links only.  Useful for retrieving a specific
home page without any distractions, not even those
from the same hosts.</string>
             </property>
             <property name="text">
              <string>Follow Relative Links Only</string>
             </property>
            </widget>
            <widget class="QCheckBox" name="cbNoParentOBP">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>130</y>
               <width>105</width>
               <height>25</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>Do not ever ascend to the parent directory when
retrieving recursively.  This is a useful option, since it
guarantees that only the files below a certain hierarchy
will be downloaded.</string>
             </property>
             <property name="text">
              <string>No Parent</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageFTP">
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Customize FTP project</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>FTP Url</string>
              </property>
              <property name="buddy">
               <cstring>leFTPURL</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leFTPURL">
              <property name="toolTip">
               <string>Use special characters (wildcards), like *, ?, [ and ] to
retrieve more than one file from the same directory at
once.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="cbRetrieveSimLinkFTP">
            <property name="toolTip">
             <string>When retrieving FTP directories recursively and a 
symbolic link is encountered, the linked-to file is not 
downloaded.
Use this to enable retrieving for symbolic links.</string>
            </property>
            <property name="text">
             <string>Retrieve Symbolic Links</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Level depth</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="sbLevelDepthFTP"/>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QGroupBox" name="gbUseAcc">
            <property name="title">
             <string>Use Account</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Username</string>
               </property>
               <property name="buddy">
                <cstring>leUser</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="leUser">
               <property name="toolTip">
                <string>FTP Username</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Password</string>
               </property>
               <property name="buddy">
                <cstring>lePass</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lePass">
               <property name="toolTip">
                <string>FTP Password</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageCustom">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Custom Project</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>500</width>
               <height>220</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0" colspan="2">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Download The following files: (File types)</string>
                </property>
                <property name="buddy">
                 <cstring>cbAcceptList</cstring>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="3">
               <widget class="QComboBox" name="cbAcceptList">
                <property name="editable">
                 <bool>true</bool>
                </property>
                <item>
                 <property name="text">
                  <string>Don't use list</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">rar,tar,zip,tgz</string>
                 </property>
                 <property name="icon">
                  <iconset resource="image.qrc">
                   <normaloff>:/image/icon/big/utamacompressed.png</normaloff>:/image/icon/big/utamacompressed.png</iconset>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">jpg,jpeg,png,gif,tiff,svga,bmp</string>
                 </property>
                 <property name="icon">
                  <iconset resource="image.qrc">
                   <normaloff>:/image/icon/big/Pollen_Flower.png</normaloff>:/image/icon/big/Pollen_Flower.png</iconset>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">pdf,ps</string>
                 </property>
                 <property name="icon">
                  <iconset resource="image.qrc">
                   <normaloff>:/image/icon/big/utamadocument.png</normaloff>:/image/icon/big/utamadocument.png</iconset>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">doc,xls,odt,ods,odp,odf,ppt,rtf</string>
                 </property>
                 <property name="icon">
                  <iconset resource="image.qrc">
                   <normaloff>:/image/icon/big/categoryfiles.png</normaloff>:/image/icon/big/categoryfiles.png</iconset>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">c,cpp,java,py,cxx,h,hpp,sh,css,php,pl</string>
                 </property>
                 <property name="icon">
                  <iconset resource="image.qrc">
                   <normaloff>:/image/icon/big/scriptman.png</normaloff>:/image/icon/big/scriptman.png</iconset>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>mp3,ogg,wav</string>
                 </property>
                 <property name="icon">
                  <iconset resource="image.qrc">
                   <normaloff>:/image/icon/big/utamamusic.png</normaloff>:/image/icon/big/utamamusic.png</iconset>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>mpg,mp4,avi,ogm,mkv,rm,rmvb,flv</string>
                 </property>
                 <property name="icon">
                  <iconset resource="image.qrc">
                   <normaloff>:/image/icon/big/utamavideo.png</normaloff>:/image/icon/big/utamavideo.png</iconset>
                 </property>
                </item>
               </widget>
              </item>
              <item row="2" column="0" colspan="2">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Don't download the following files:</string>
                </property>
                <property name="buddy">
                 <cstring>cbRejectList</cstring>
                </property>
               </widget>
              </item>
              <item row="3" column="0" colspan="3">
               <widget class="QComboBox" name="cbRejectList">
                <property name="editable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QCheckBox" name="cbSearchLocalOnly">
                <property name="text">
                 <string>Search file in this site Only</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Level depth</string>
                </property>
                <property name="buddy">
                 <cstring>sbLevelDepthCP</cstring>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QSpinBox" name="sbLevelDepthCP">
                <property name="toolTip">
                 <string>Set maximum recursion depth. (Use 0 to set Infinite depth)</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QCheckBox" name="cbConvertLinkCP">
                <property name="toolTip">
                 <string>After the download is complete, convert the links in the
document to make them suitable for local viewing.  This
affects not only the visible hyperlinks, but any part of
the document that links to external content, such as
embedded images, links to style sheets, hyperlinks to
non-HTML content, etc.</string>
                </property>
                <property name="whatsThis">
                 <string>After the download is complete, convert the links in the document to make them suitable for local viewing.  This affects not only the visible hyperlinks, but any part of the document that links to external content, such as embedded images, links to style sheets, hyperlinks to non-HTML content, etc.</string>
                </property>
                <property name="text">
                 <string>Convert Link</string>
                </property>
               </widget>
              </item>
              <item row="6" column="2">
               <widget class="QCheckBox" name="cbCreateBackupCP">
                <property name="text">
                 <string>Create Backup</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QCheckBox" name="cbFollowFTPCP">
                <property name="text">
                 <string>Follow FTP</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QCheckBox" name="cbNoParentCP">
                <property name="text">
                 <string>No Parent</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbMirror</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbConvertLinkOBP</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbCreateBackupOBP</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbConvertLinkOBP</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbCreateBackupOBP</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbConvertLinkCP</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbCreateBackupCP</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbConvertLinkCP</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbCreateBackupCP</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
